### PR TITLE
Arr/add user to circle functionality

### PR DESCRIPTION
**Pull request by Statmosphere**
**Changes**
Include a short description of your changes

**Notes**
Include any notes you have about your changes, or for the reviewer

**YouTrack Link**
Include the link to the corresponding YouTrack item

### CODE REVIEW

File: package.json
Comment (aidan-hubley) [inline package.json:1]: Why is this here? Pretty sure we don't want this package...

File: app/(settings)/circleSettings.js
Comment (aidan-hubley) [inline app/(settings)/circleSettings.js:1]: What happened here? Why is this file edited at all?

**Review by aidan-hubley — CHANGES_REQUESTED** [review]
Overall it's great, although I don't know why circleSettings was changes... I also don't think we should add that SVG package back...

File: package.json
Comment (Statmosphere) [reply package.json:1]: It was added when my app wouldn't start. I've removed it and I was able to access the app fine without it.

File: app/(settings)/circleSettings.js
Comment (Statmosphere) [reply app/(settings)/circleSettings.js:1]: Leftovers from another branch. I'll probably use this in the future but for the purposes of this pull request I'll remove it.

**Review by Statmosphere — COMMENTED** [review]
Made all of the changes and added both reloading and checking to see if you're already a member.

**Review by aidan-hubley — APPROVED** [review]
Well done! Looking forward to public codes adding a user to the member queue!

**Pull request merged**